libs/wingsdk/src/platform/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./platform";
+export * from "./platform-manager";
```

Comment:
I dont have a great reason for putting these in their own namespace, might be worth just including it in core